charts/timed/templates/configmap-backend.yaml

```diff
@@ -53,10 +53,10 @@ data:
 {{- end }}
   DJANGO_OIDC_ADMIN_LOGIN_REDIRECT_URL: {{ .Values.auth.oidc.adminLoginRedirectUrl | quote }}
 
-{{- if .Values.postgresql.enabled }}
+{{- if not .Values.postgresql.existingHost }}
```

Comment:
i'd suggest something like this

```
{{- if .Values.postgresql.enabled }}
  DJANGO_DATABASE_HOST: {{ include "timed.postgresql.fullname" . | quote }}
{{- else if .Values.postgresql.existingHost }}
  DJANGO_DATABASE_HOST: {{ .Values.postgresql.existingHost | quote }}
{{- else }}
  DJANGO_DATABASE_HOST: {{ .Values.postgresql.service.host | quote }}
{{- end }}
```

An ExternalAddress would probably make this a bit less confusing overall.